test/extended/cluster/utils.go

```diff
@@ -519,3 +519,39 @@ func SetNamespaceLabels(c kclientset.Interface, name string, labels map[string]s
 	ns.Labels = labels
 	return c.CoreV1().Namespaces().Update(ns)
 }
+
+func ProjectExists(oc *exutil.CLI, name string) (bool, error) {
+	projectList, err := oc.AdminProjectClient().Project().Projects().List(metav1.ListOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	for _, p := range projectList.Items {
+		if name == p.Name {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func DeleteProject(oc *exutil.CLI, name string, sync bool, interval, timeout time.Duration) error {
+	e2e.Logf("Deleting project %v ......", name)
+	err := oc.AdminProjectClient().Project().Projects().Delete(name, &metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	if sync {
+		err = wait.Poll(interval, timeout, func() (bool, error) {
```

Comment:
Jiri has written his own custom polling to see if the namespace exists, but there are many existing function in the extended/e2e framework that do this as well. Can we please not do it ourselves again?

Reply:
The closest thing is e2e framework is [WaitUntilPodIsGone](https://github.com/openshift/origin/blob/master/test/extended/util/framework.go#L1056).

Would be nice to have `WaitUntilProjectIsGone`. Maybe I should do it there.